src/client.js

```diff
@@ -6,6 +6,7 @@ import { CommandoClient } from 'discord.js-commando'
 const {
   OWNERS_IDS = '269617876036616193', // Default to @evan#9589
   COMMAND_PREFIX = '!',
+  NODE_ENV = 'development',
```

Comment:
It is dynamic, this is just ensuring the default is set to `development`. If `NODE_ENV` is set to `production` then it will not be overriden!

Try it yourself!

```js
// node-env.js
const { NODE_ENV = 'development' } = process.env
console.log(NODE_ENV)
```

```
[sustained@aurora Playground]$ NODE_ENV=production node node-env.js
production
```

:smile: